karapace/compatibility/__init__.py

```diff
@@ -53,45 +63,74 @@ def check_jsonschema_compatibility(reader: Draft7Validator, writer: Draft7Valida
 
 
 def check_compatibility(
-    source: TypedSchema, target: TypedSchema, compatibility_mode: CompatibilityModes
```

Comment:
I did this change mostly because of these names, I always had to double check what was the source and target schemas.